Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM talis/ubuntu:1404-latest
```

Comment:
FYI, This was causing build errors locally which were needed for tests. This is only needed for local dev.